services/src/northbound-service/northbound/src/main/java/org/openkilda/northbound/utils/RequestCorrelationFilter.java

```diff
@@ -48,7 +48,11 @@
     private static final Logger logger = LoggerFactory.getLogger(RequestCorrelationFilter.class);
 
     private static final List<String> EXCLUDE_PATTERNS = ImmutableList.of(
-            "/v1/health-check"
+            "/v1/health-check",
+            // swagger related patterns
+            "/swagger*/**",
```

Comment:
nit: there are the same exclude patterns for RequestCorrelationFilter as well, Maybe it's worth to move it in constants